geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/jts/udf/SpatialRelationFunctionsTest.scala

```diff
@@ -485,6 +485,25 @@ class SpatialRelationFunctionsTest extends Specification with TestEnvironment {
       dfBlank.select(st_length(lit(null))).first must beNull
     }
 
+    "st_translate" >> {
+      val expected = WKTUtils.read("LINESTRING(1 2, 11 2)")
+      val trans = dfBlank.select(st_translate(st_geomFromWKT("LINESTRING(0 0, 10 0)"), 1, 2)).first
+      trans mustEqual expected
+    }
+
+    "st_aggregateDistanceSpheroid" >> {
+      val p1 = points("int")
+      val p2 = points("edge")
+      val dist = dfBlank.select(st_aggregateDistanceSpheroid(array(st_geomFromWKT(p1), st_geomFromWKT(p2)))).first
+      dist.toDouble must beGreaterThan(550000.0)
+    }
+
+    "st_lengthSpheroid" >> {
+      val line = "LINESTRING(1 2, 11 2)"
+      val dist = dfBlank.select(st_lengthSpheroid(st_castToLineString(st_geomFromWKT(line)))).first
+      dist.toDouble must beGreaterThan(1110000.0)
```

Comment:
This and the other "magic number" above were extracted from printing the results, assuming that the math behind them is correct. Test case is just to exercise the call path.

Reply:
hmm, instead of using the magic number that isn't verified, maybe just change to something like
`dfBlank.select(st_lengthSpheroid(st_castToLineString(st_geomFromWKT(line)))).first must not(throwAn[Exception])`
that way it's clear that we aren't actually validating the output.